docker-compose.yml

```diff
@@ -35,7 +35,7 @@ services:
       dockerfile: docker/Dockerfile.dev
     image: 311462405659.dkr.ecr.eu-west-1.amazonaws.com/serve-opg/api:latest
     ports:
-      - 9000:9000
+      - 9090:9090
```

Comment:
I'd be surprised if you need this. This exposes your external port 9090 to internal port 9090 (on the docker network). As nginx will be routing to internal to the network and same for the app, it shouldn't need to be exposed. It's not a problem, but don't think it's needed :)

Reply:
I did this to deconflict with the app being on 9000 as well locally. The API doesn't actually get deployed yet anyway

Reply:
Ah just seen your other comment, I think I'll leave it for now as it doesn't break anything